### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tajrink**
tajrin kashem
@rivlev  @andreas-weise

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# hello-world
\ No newline at end of file
+# hello-world
```
Comment (rivlev) [inline README.md:1]: this is not a comment, this is a preprocessor directive

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
...
+#include <iostream>
+  using namespace std;
```
Comment (rivlev) [inline README.md:3]: no indentation

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
...
+  int main()
+  {
+  cout<<"hello world"<<endl;
+  return 0;
+  }
```
Comment (rivlev) [inline README.md:8]: not called helloworld.cpp

Comment (rivlev) [reply README.md:8]: not cloned properly through the schedule link to github classroom

Comment (rivlev) [reply README.md:8]: 4/10

**Comment by rivlev** [comment]
Please close this pull request (credit not assigned until pull request is closed)

**Comment by rivlev** [comment]
You were not able to activate travis ci because you did not create the project properly through github classroom and it is not part of the cisc3142 organization

**Comment by tajrink** [comment]
> You were not able to activate travis ci because you did not create the project properly through github classroom and it is not part of the cisc3142 organization

File: README.md (hunk 3 above)
Comment (tajrink) [reply README.md:8]: hi, Professor how can I resolve this issue?

**Pull request closed without merging**